react/headless/src/config.tsx

```diff
@@ -3,14 +3,13 @@ import {createWidgetsConfig, type WidgetsConfigStore, type WidgetsConfig as Core
 import {computed} from '@amadeus-it-group/tansu';
 import type {ReactNode} from 'react';
 import {createContext, useContext, useEffect, useMemo} from 'react';
-import type {AdaptPropsSlots} from './types';
 import {useWidget} from './utils/widget';
 import {usePropsAsStore} from './utils/stores';
 
 export * from '@agnos-ui/core/config';
 
 export type WidgetsConfig = {
```

Comment:
It looks like this type is now an exact (verbose) copy of `CoreWidgetsConfig`. You could simply remove it, right?